### PR TITLE
WIP: Issue_#68: add 'verification steps' to the procedure module

### PR DESCRIPTION
**Pull request by tylerkelly13**
Draft resolution to Issue #68 . 
Feel free to suggest/discuss changes.

### CODE REVIEW

File: modular-docs-manual/content/topics/module_guidelines-procedure.adoc [hunk 1]
```diff
@@ -30,6 +30,13 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+[discrete]
+== Procedure Verification Steps
+This section is recommend but optional. Provide the user one or more steps to verify that the procedure provided the intended outcome. This may consist of:
```
Comment (bhardesty) [inline modular-docs-manual/content/topics/module_guidelines-procedure.adoc]: recommend -> recommended

Provide the user _with_...

Comment (bhardesty) [reply modular-docs-manual/content/topics/module_guidelines-procedure.adoc]: I agree that this section should be optional, but I don't think we should recommend it. In many cases, the verification should just be the final step (or steps) in the procedure.

File: modular-docs-manual/content/topics/module_guidelines-procedure.adoc [hunk 2]
```diff
@@ -30,6 +30,13 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+[discrete]
+== Procedure Verification Steps
```
Comment (ncbaratta) [inline modular-docs-manual/content/topics/module_guidelines-procedure.adoc]: This should be sentence case per the new guidelines for headings

File: modular-docs-manual/content/topics/module_guidelines-procedure.adoc (hunk 1 above)
Comment (tylerkelly13) [reply modular-docs-manual/content/topics/module_guidelines-procedure.adoc]: Thanks for spotting the typos :) I was in too much of a hurry.
Removed "... recommended but ..."

File: modular-docs-manual/content/topics/module_guidelines-procedure.adoc (hunk 2 above)
Comment (tylerkelly13) [reply modular-docs-manual/content/topics/module_guidelines-procedure.adoc]: Thanks @ncbaratta . I have changed the heading to sentence case. Note that the rest of the doc is still in title case though :)

**Comment by tylerkelly13** [comment]
Does anyone have any other examples of the types of verification steps we can include here? Both for clarity and to give other writers some ideas? Much better to have verification steps too often in our docs than not often enough. As a user, I would not be happy if I followed a series of procedures, only to find I unknowingly made an error in an early procedure.

**Comment by bhardesty** [comment]
I typically include a verification step (or steps) as the final steps in the procedure itself. Here's a simple example: https://access.redhat.com/documentation/en-us/red_hat_amq/7.2/html-single/using_amq_interconnect/#starting-the-router

More complex procedures will likely require more complex verification.

**Comment by tylerkelly13** [comment]
@bhardesty thanks for the comment. Do you believe there are any changes needed? Writers or teams will decide if it is necessary to distinguish between the procedure and verification steps (or not).

File: modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc [hunk 3]
```diff
@@ -29,6 +29,9 @@ This paragraph is the procedure module introduction: a short description of the
 
 . Use an unnumbered bullet (*) if the procedure includes only one step.
 
+//.Verification steps
```
Comment (finp) [inline modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc]: @tylerkelly13 looks good, just curious, why is it commented?

**Comment by adahms** [comment]
Resolved all open discussions after checking that feedback had been incorporated and there were no outstanding items.

**Comment by adahms** [comment]
There have been no additional comments or requests on this PR since the last comment; merging.

**Pull request merged**